submariner/fine-grained-connectivity.md

```diff
@@ -0,0 +1,86 @@
+# Fine Grained Connectivity
+
+## Summary
+
+Submariner currently provisions full mesh topology by default between the participating clusters.
+For some use-cases, we need fine-grained connectivity options. This proposal covers only the topology
+aspect of connectivity.
+
+This proposal aims to address the below use-case
+
+Hub-Spoke Topology -  Here only the client can talk to the server but not to the other clients to
+which the server is connected to.
+
+To achieve each hub and spoke will be grouped together into an isolated group of clusters called Clusterset.
```

Comment:
I find it confusing that we include design details ("each hub and spoke will be grouped together into an isolated group of clusters called Clusterset") as part of the use-case definition. I think it's better to only define the problems/requirements in this section, and explain the proposed approach under the Proposal section below.